docs/aws-config.md

```diff
@@ -147,6 +147,10 @@ Specify the following values in the example files:
 
   * Replace `<YOUR_REGION>`. See the [VolumeSnapshotLocation definition][6] for details.
 
+* (Optional, use only if you need to specify multiple volume snapshot locations) In `examples/aws/10-deployment.yaml` (or `examples/aws/10-deployment-kube2iam.yaml`, as appropriate):
```

Comment:
s/need to specify/have

Reply:
or omit altogether ("use to specify multiple ...). Will fix up when the rest is done.